providers/fcontext.rb

```diff
@@ -25,6 +25,23 @@ def restorecon(file_spec)
   return "restorecon -iR #{path}"
 end
 
+def semanage_options(filetype)
+  # Set options for filetype
+  if filetype
```

Comment:
OK, see my comment below. If we actually keep `filetype` as `nil` by default and this works, then my fears are irrelevant (I was afraid we can get `""` somehow here)